src/Uno.UI/UI/Xaml/Media/LinearGradientBrush.wasm.cs

```diff
@@ -23,5 +24,26 @@ internal string ToCssString(Size size)
 
 			return $"linear-gradient({angle}rad,{stops})";
 		}
+
+		/// <summary>
+		/// Generates a linearGradient element that can be used inside SVG-based views (Path, etc)
+		/// </summary>
+		internal UIElement ToSvgElement()
+		{
+			var linearGradient = new SvgElement("linearGradient");
+
+			linearGradient.SetAttribute(
+				("x1", StartPoint.X.ToString()),
+				("y1", StartPoint.Y.ToString()),
+				("x2", EndPoint.X.ToString()),
+				("y2", EndPoint.Y.ToString())
```

Comment:
They need the `Invariant` number format.

Reply:
fixed